sinon/lib/spy.py

```diff
@@ -57,7 +57,7 @@ def __remove_args_first_item(self):
         if len(self.args) > 0:
             new_args_list = []
             for item in self.args:
-                if self.obj == item[0].__class__:
+                if (len(item) > 0) and (self.obj == item[0].__class__):
```

Comment:
Here is no problem. But I still want to tell you one thing about Python. In python, if statement should be simple without brackets if not necessary. Some pythonic coding style is not easy to learn from other language. Hope you can enjoy this language's philosophy .

In this case, if you use [pylint](https://google.github.io/styleguide/pyguide.html) to check, you will get this message `C: 60, 0: Unnecessary parens after 'if' keyword (superfluous-parens)`.

To use pylint, just `pip install pylint` and run `pylint <target file path>`

Reply:
It would be nice if the continuous integration build ran pylint

Reply:
Actually, [this guy](https://landscape.io/github/note35/sinon/62) check that. 💪